src/qt/pivx/receivewidget.cpp

```diff
@@ -338,7 +338,14 @@ void ReceiveWidget::sortAddresses()
 
 void ReceiveWidget::onTransparentSelected(bool transparentSelected)
 {
-    this->shieldedMode = !transparentSelected;
+    shieldedMode = !transparentSelected;
+
+    if (shieldedMode && !walletModel->isSaplingEnforced()) {
+        ui->pushLeft->setChecked(true);
+        inform(tr("Cannot perform shielded operations, v5 upgrade hasn't being enforced yet!"));
```

Comment:
change to either `hasn't been enforced` or `isn't being enforced`

Reply:
typo fixed